.github/workflows/gradle.yml

```diff
@@ -11,35 +11,22 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [1.8, 11]
+        java: [8, 11]
         os: [ubuntu-latest, windows-latest, macos-latest]
     runs-on: ${{ matrix.os }}
-    env:
-      gradle_version: 6.8.3  # set to empty to build with most recent version
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v1
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK ${{ matrix.java }} and Gradle
+        uses: ome/action-gradle@v3.2
         with:
           java-version: ${{ matrix.java }}
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-          restore-keys: ${{ runner.os }}-gradle
-      - name: Wrap with specified version
-        run: gradle wrapper --gradle-version=${{ env.gradle_version }}
-        if: ${{ env.gradle_version != '' }}
-      - name: Wrap without version
-        run: gradle wrapper
-        if: ${{ env.gradle_version == '' }}
       - name: Prepare minio
         run: |
           (cd ci && ./startminio.sh)
           echo AWS_ACCESS_KEY_ID=stsadmin >> $GITHUB_ENV
           echo AWS_SECRET_ACCESS_KEY=stsadmin-secret >> $GITHUB_ENV
       - name: Build
         run: |
-          gradle check
+          gradle check -PexcludeTests="**/s3*" 
```

Comment:
:+1: and we can come back to this later. You might want to comment/remove `startminio.sh`, too, if things keep failing.

Reply:
if I cannot figure out the reason,  I will remove the ``minio`` section as mentioned. I keep it for now